pythonforandroid/build.py

```diff
@@ -605,23 +600,16 @@ def build_recipes(build_order, python_modules, ctx, project_dir,
         ignore_setup_py=ignore_project_setup_py
     )
 
-    return
-
 
 def project_has_setup_py(project_dir):
-    if project_dir is not None and \
-            (os.path.exists(os.path.join(project_dir,
-                            "setup.py")) or
-             os.path.exists(os.path.join(project_dir,
-                            "pyproject.toml"))
-            ):
-        return True
-    return False
+    return project_dir is not None and \
```

Comment:
**nitpick:** this backslash hurts my eyes :laughing:, since we touch the lines we could remove that and do it more :snake:.

Reply:
Thanks addressed :+1: